test/StdChains.t.sol

```diff
@@ -3,6 +3,24 @@ pragma solidity >=0.7.0 <0.9.0;
 
 import "../src/Test.sol";
 
+contract StdChainsMock is Test {
+    function getChain_(string memory chainAlias) public returns (Chain memory) {
```

Comment:
wdyt about `exposed_getChain`, etc for all methods? that's what we have recommended in the forge docs for tests of this style

Reply:
sounds good! will update here and in forge-std